### PR TITLE
Update README. Upd

### PR DESCRIPTION
**Pull request by r0gergunn**
Desc

### CODE REVIEW

**Comment by yorlov** [comment]
Regular comment

**Comment by r0gergunn** [comment]
Comment

**Comment by yorlov** [comment]
Comment

**Comment by yorlov** [comment]
Comment 3

**Comment by yorlov** [comment]
Comment 4

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # Play Framework - The High Velocity Web Framework
 
-[![Gitter](https://img.shields.io/gitter/room/gitterHQ/gitter.svg)](https://gitter.im/playframework/playframework?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [<img src="https://img.shields.io/travis/playframework/playframework.svg"/>](https://travis-ci.org/playframework/playframework) [![Maven](https://img.shields.io/maven-central/v/com.typesafe.play/play_2.13.svg)](http://mvnrepository.com/artifact/com.typesafe.play/play_2.13)
```
Comment (yorlov) [inline README.md:3]: Hm

File: README.md [hunk 2]
```diff
@@ -1,7 +1,5 @@
...
-
 The Play Framework combines productivity and performance making it easy to build scalable web applications with Java and Scala.  Play is developer friendly with a "just hit refresh" workflow and built-in testing support.  With Play, applications scale predictably due to a stateless and non-blocking architecture.  By being RESTful by default, including assets compilers, JSON & WebSocket support, Play is a perfect fit for modern web & mobile applications.
 
 ## Learn More
```
Comment (yorlov) [inline README.md:5]: Oops

**Review by yorlov — CHANGES_REQUESTED** [review]
Yo

File: README.md (hunk 2 above)
Comment (yorlov) [reply README.md:5]: reply

**Review by yorlov — APPROVED** [review]
Ik

File: README.md (hunk 1 above)
Comment (yorlov) [reply README.md:3]: asdasd

**Comment by yorlov** [comment]
Regular

**Comment by yorlov** [comment]
Re 2

**Review by yorlov — COMMENTED** [review]
fd

**Comment by yorlov** [comment]
Comment. Upd 4

**Comment by yorlov** [comment]
Hi

File: README.md (hunk 1 above)
Comment (yorlov) [inline README.md:3]: Hm. Upd

Comment (r0gergunn) [reply README.md:3]: Reply. Upd 2

Comment (yorlov) [reply README.md:3]: No. Upd

Comment (r0gergunn) [inline README.md:3]: ye?

Comment (yorlov) [reply README.md:3]: ok

File: README.md [hunk 3]
```diff
@@ -1,7 +1,5 @@
 # Play Framework - The High Velocity Web Framework
```
Comment (yorlov) [inline README.md:1]: ZY

**Review by yorlov — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (r0gergunn) [reply README.md:1]: p

**Pull request closed without merging**